docs/configuration/prune.rst

```diff
@@ -219,6 +219,33 @@ Policy ``regex``
 Like all other regular expression fields in prune policies, zrepl uses Go's `regexp.Regexp <https://golang.org/pkg/regexp/#Compile>`_ Perl-compatible regular expressions (`Syntax <https://golang.org/pkg/regexp/syntax>`_).
 The optional `negate` boolean field inverts the semantics: Use it if you want to keep all snapshots that *do not* match the given regex.
 
+.. _prune-filters:
+
+Filesystem filters in pruning policies
+--------------------------------------
+:ref:`Filesystem filters<pattern-filter>` can be used to define different pruning policies within a single job through the ``filesystems`` field. If omitted, it defaults to ``{ "<": true }`` (that is, all filesystems are matched).
+
+Example configuration:
+
+::
+
+    jobs:
+    - type: pull
+      pruning:
+        keep_receiver:
+        - type: grid
+          filesystems:
+            "tank/long": true
```

Comment:
Are these filesystem names sender-side or receiver side?

Regardless, please provide the `root_fs` of this `pull` job in the example to clarify.

Reply:
Hope I got it right - didn't include an explicit `root_fs` but mirrored my tested config with a similar format to existing docs